docs/pages/installation.mdx

```diff
@@ -485,6 +485,9 @@ $ kubectl exec -i my-pod -- tctl status
 # sending local files via stdin
 $ kubectl exec -i my-pod -- tctl create -f < my-local-file.yaml
 
+# retrieving the teleport service config file from the configmap
+$ kubectl get configmap teleport-cluster-auth -o jsonpath="{.data['teleport\.yaml']}"
```

Comment:
It looks like this only refers to the Auth Service configmap in the `teleport-cluster` chart. Do we have a user-friendly way to list all available configmaps first? E.g., `kubectl get configmaps` in the appropriate namespace? Or we could include similar example commands for the Proxy Service in `teleport-cluster` and agents in `teleport-kube-agent`.

Reply:
The other examples seemed like they were assuming some knowledge but i'm an advocate for adding more details. @zmb3 do you think this will bloat things too much or should we add them in?

Reply:
I think this is good enough to merge. We're just including a few snippets for how to accomplish common tasks in this section, not trying to teach the ins and outs of how to use kubectl.